src/CMakeLists.txt

```diff
@@ -452,6 +452,18 @@ target_link_libraries(ceph-common ${ceph_common_deps})
 set_target_properties(ceph-common PROPERTIES
   SOVERSION 0
   INSTALL_RPATH "")
+if(NOT APPLE)
+  # Apple uses Mach-O, not ELF. so this option does not apply to APPLE.
+  #
+  # prefer the local symbol definitions when binding references to global
+  # symbols. otherwise we could reference the symbols defined by the application
+  # with the same name, instead of using the one defined in libceph-common.
+  set_property(
```

Comment:
Maybe restate comment? Eg: "Require libceph-common to use local symbols, even if redefined in application".

Reply:
sure, will do.